caretaker/src/components/background.js

```diff
@@ -0,0 +1,58 @@
+import React, { PureComponent } from 'react'
+import styled from 'styled-components'
+import mq from '../utils/media-query'
+import mobileBgLeft from '../../static/nursing/background-mobile-left.png'
+import tabletBgLeft from '../../static/nursing/background-tablet-left.png'
+import desktopBgLeft from '../../static/nursing/background-desktop-left.png'
+import hdBgLeft from '../../static/nursing/background-hd-left.png'
+import mobileBgRight from '../../static/nursing/background-mobile-right.png'
+import tabletBgRight from '../../static/nursing/background-tablet-right.png'
+import desktopBgRight from '../../static/nursing/background-desktop-right.png'
+import hdBgRight from '../../static/nursing/background-hd-right.png'
+
+const Container = styled.div`
+  position: absolute;
+  top: 0;
+  left: 0;
+  display: flex;
+  align-items: stretch;
+  height: 100%;
+  width: 100%;
+`
+
+const Column = styled.div`
+  background-size: contain;
+  ${mq.mobileOnly`
+    flex: 0 0 20px;
+    background-image: url(${props => (props.reversed ? mobileBgRight : mobileBgLeft)});
+  `}
+  ${mq.tabletOnly`
+    flex: 0 0 32px;
+    background-image: url(${props => (props.reversed ? tabletBgRight : tabletBgLeft)});
+  `}
+  ${mq.desktopOnly`
+    flex: 0 0 32px;
+    background-image: url(${props => (props.reversed ? desktopBgRight : desktopBgLeft)});
+  `}
+  ${mq.hdAbove`
+    flex: 0 0 40px;
+    background-image: url(${props => (props.reversed ? hdBgRight : hdBgLeft)});
+  `}
+`
```

Comment:
It seems that adding `z-index` value bigger than `40`(which is the `z-index` value of `slidesContainerFocus`) will fix the disappeared opening column issue.

Reply:
My fault! Modify the `z-index` will lead to `PopUp` button being hidden by `Column`.
I created a [PR](https://github.com/YuCJ/static-fe-boilerplate/pull/1) to fixe this issue.

Reply:
Thanks for solving the problem. 🙏  I've merged the PR.